scoring-mp/test/addContainer.sh

```diff
@@ -5,10 +5,10 @@ echo -e " \e[32m@@@ Excuting script: \e[1;33maddContainer.sh \e[0m"
 
 ## Variables
 
-KAFKA_BROKERS=""
-KAFKA_APIKEY=""
-CA_LOCATION=""
-PEM_FILE=""
+KAFKA_BROKERS="eventstream140-ibm-es-proxy-route-bootstrap-eventstreams.apps.green.ocp.csplab.local:443"
```

Comment:
Shouldn't we avoid pushing these credentials to github?

Reply:
Good catch, somehow those got in there with the `git add`...  

Removed them and will recreate API key now.